src/pages/Home.js

```diff
@@ -0,0 +1,11 @@
+import Shared from "./Shared";
+
+const Home = () => {
+  return (
+    <>
+      <Shared></Shared>
+    </>
```

Comment:
`Fragment` `<></>`는 필요가 없습니다. 그냥 컴포넌트를 바로 리턴시켜주어도 됩니다.
```suggestion
      <Shared></Shared>
```